cluster and persistence

patriknw opened the pull request.

johanandren commented on src/draft/blog.md:
````diff
@@ -0,0 +1,90 @@
+val cluster = Cluster(system) 
+```
+
+The typed `Cluster` extension gives access to two management tasks (joining, leaving, downing, ...) and subscription of cluster membership events (MemberUp, MemberRemoved, UnreachableMember, ...). Those are exposed as two different actor references, i.e. it's a message based API. For example joining can be done like this:
````
Mark the events as code?

johanandren commented on src/draft/blog.md:
````diff
@@ -0,0 +1,90 @@
+There are also some accessors to current cluster state, such as `cluster.selfMember` and `cluster.state`.
+```
+cluster.selfMember
+```
````
Maybe that is self evident enough from the sentence above without any sample code block?

johanandren commented on src/draft/blog.md:
````diff
@@ -0,0 +1,90 @@
+TODO Routee code
+```
+
+Pretty amazing functionality in 20 lines of code.
````
Pretty amazing! :)

johanandren commented on src/draft/blog.md:
````diff
@@ -0,0 +1,90 @@
+```
+
+Another thing that is good to know is that the same [serialization](http://doc.akka.io/docs/akka/current/scala/serialization.html) mechanism for messages as for untyped actors is also used in Akka Typed. Actor references are typically included in the messages, since there is no `sender`. To serialize actor references to/from string representation you will use the `ActorRefResolver` extension as i ...
+
````
How about splitting it here, typed cluster intro, and then the rest below typed cluster tools and a separate one for persistence? Written stuff on-the-line vs in books and attention spans etc.

johanandren commented on src/main/scala/blog/typed/cluster/scaladsl/RandomRouter.scala:
```diff
@@ -0,0 +1,91 @@
+        }
+
+      routingBehavior(Vector.empty)
+    }.narrow[T]
```
Pretty amazing indeed!

patriknw commented:
incorporated Johan's feedback and created PR for the actual blog post in https://github.com/akka/akka.github.com/pull/463

Added more examples for ddata and singleton, which will be in next blog post.
